pyiceberg/io/pyarrow.py

```diff
@@ -1216,6 +1216,25 @@ def _field_id(self, field: pa.Field) -> int:
         return -1
 
 
+def _get_column_projection_values(
+    file: DataFile,
+    projected_schema: Schema,
+    projected_field_ids: Set[int],
+    file_project_schema: Schema,
```

Comment:
Since we only need the IDs, I'd rather just pass in those for clarity:
```suggestion
    file_project_schema: Set[int],
```